stubs/docutils/@tests/stubtest_allowlist.txt

```diff
@@ -7,16 +6,17 @@ docutils.io.FileOutput.__init__
 docutils.io.Input.__init__
 docutils.languages.LanguageImporter.__getattr__
 docutils.nodes.Element.__getattr__
-docutils.nodes.GenericNodeVisitor.__getattr__
 docutils.nodes.Element.__iter__ # doesn't exist at runtime, but the class is iterable due to __getitem__
+docutils.nodes.GenericNodeVisitor.__getattr__
```

Comment:
sorry about the churn in this file. It was *almost* in alphabetical order, which was painful to look at